### ISSUE TITLE
Add attributes to the notification list

### ISSUE THREAD

**Issue by demery** [demery/iiif-ldn-demo#7]
Add these notification attributes to the list of notifications returned by `GET /iiif/notifications`: `updated`, `source`. Also, default `motivation` to `iiifsupplment`.

Sample:

```json-ld
{
  "@context": "http://www.w3.org/ns/ldp",
  "@id": "http://sims-dev.library.upenn.edu:8084/iiif/notifications",
  "contains": [
    {
      "url": "http://sims-dev.library.upenn.edu:8084/iiif/notifications/58cc2e0d08ff803627181ea9",
      "motivation": "iiifsupplement",
      "updated": "2017-03-17 01:34:24 UTC",
      "source": "http://scta.info"
    },
    {
      "url": "http://sims-dev.library.upenn.edu:8084/iiif/notifications/58cc2e0d08ff803627181eaa",
      "motivation": "metadata",
      "updated": "2017-03-17 01:34:24 UTC",
      "source": "http://scta.info"
    }
  ]
}
```

**Comment by kelynch** [comment]
Addressed by #11

**Comment by kelynch** [comment]
Closing.
